src/vcpkg/archives.cpp

```diff
@@ -67,6 +67,26 @@ namespace vcpkg::Archives
                                code_and_output.output);
             recursion_limiter_sevenzip_old = false;
         }
+        else if (ext == ".msi")
+        {
+            // msiexec is a WIN32/GUI application, not a console application and so needs special attention to wait
```

Comment:
I would prefer to see cmd_execute_and_capture_output be taught to do that (by waiting on the process handle) but isn't a huge deal given that N is 1.

Reply:
Agreed when N>1